app/assets/javascripts/patches/ember-cli+4.12.0.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/ember-cli/lib/models/project.js b/node_modules/ember-cli/lib/models/project.js
+index b21a9bcd4..6537fcc2e 100644
+--- a/node_modules/ember-cli/lib/models/project.js
++++ b/node_modules/ember-cli/lib/models/project.js
+@@ -409,6 +409,7 @@ class Project {
+     @return {Object}       Bower dependencies
+    */
+   bowerDependencies(bower) {
++    return {};
```

Comment:
Something like this so that it's obvious what we're doing?
```suggestion
+    return {}; // Silence deprecation until Discourse is on Ember 4.x
```